File: lab1/python/coordinate_geometry.py
import math

point1_x = int(input("Enter for point 1 x: "))
point1_y = int(input("Enter for point 1 y: "))

point2_x = int(input("Enter for point 2 x: "))
point2_y = int(input("Enter for point 2 y: "))

result = math.sqrt((point1_x - point2_x)**2 + (point1_y - point2_y)**2)
print(result)

File: cpp/coordinate_geometry.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    int point1_x, point1_y, point2_x, point2_y;

    cout<<"Enter for point 1 x: "<<endl;
    cin>>point1_x;
    cout<<"Enter for point 1 y: "<<endl;
    cin>>point1_y;
    cout<<"Enter for point 2 x: "<<endl;
    cin>>point2_x;
    cout<<"Enter for point 2 y: "<<endl;
    cin>>point2_y;

    double result = sqrt(pow(point2_x - point1_x, 2) + pow(point2_y - point1_y, 2));

    cout<<result<<endl;

    return 0;
}